browser/css/jsdialogs.css

```diff
@@ -14,6 +14,11 @@
 	display: flex;
 }
 
+.jsdialog.ui-treeview-body {
+	min-height: 300px;
+	max-height: none;
+}
+
```

Comment:
This will affect all .ui-treeview-body widgets everywhere. Don't you want to make it specific to the container's id ?

Reply:
@pedropintosilva  Yes you are right, and I have updated the commit and have tested, this code works for me. Kindly review it now